RFS/src/main/java/com/rfs/common/http/ConnectionContext.java

```diff
@@ -85,9 +92,24 @@ public static interface Params {
 
         String getPassword();
 
+        String getSigv4Region();
+
+        String getSigv4ServiceName();
+
         boolean isInsecure();
     }
 
+    @Builder
+    @Getter
+    public static class ConstParams implements Params {
+        private String host;
+        private String username;
+        private String password;
+        private String sigv4Region;
+        private String sigv4ServiceName;
+        private boolean insecure;
+    }
```

Comment:
Pretty sure this is unneeded if we require the context be created from Params

Reply:
This is used in unit tests. Alternatively i can move this to a TestFixtures